crates/wpilib/src/encoder.rs

```diff
@@ -28,9 +28,8 @@ License version 3 as published by the Free Software Foundation. See
 <https://www.gnu.org/licenses/> for a copy.
 */
 
-// TODO: documentation
+//! HAL-based encoders.
```

Comment:
What is a HAL-based encoder?

Reply:
An encoder off the HAL? 🙃

Reply:
Honestly this seems worse than having nothing here. This documentation will have _negative_ value for teams.